wallet/src/wallet/mod.rs

```diff
@@ -443,6 +440,13 @@ impl<B: storage::Backend> Wallet<B> {
         account.get_transaction_list(skip, count)
     }
 
+    pub fn get_transactions_to_be_broadcasted(&self) -> WalletResult<Vec<SignedTransaction>> {
```

Comment:
I think it's "to be broadcast" rather than "to be broadcasted" in English.

Reply:
fixed